app/helpers/application_helper.rb

```diff
@@ -40,7 +40,7 @@ def format_period(period, style: :short)
 
   def format_in_zone(time_or_string, format: '%d %B %Y')
     time_or_string = DateTime.parse(time_or_string) if time_or_string.is_a?(String)
-    l(time_or_string.in_time_zone(LONDON_TIMEZONE), format:)
+    l(time_or_string.in_time_zone(LONDON_TIMEZONE), format:) if time_or_string
```

Comment:
Since the variable is called `time_or_string`, and the code on line 42 is based on the assumption that its value is either a time or a string, it might be neater to use a guard clause at the start to cancel out early if `time_or_string` _isn't_ a time or string but is in fact nil. But that's just a nit.

Reply:
yeah, i think that is neater 👍 -  done now